corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js

```diff
@@ -46,9 +46,53 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
         },
     });
 
+    var getGridAttributes = function (tile) {
+        if (!tile) {
+            return null;
+        }
+        var rowStart = tile.gridY + 1;
+        var colStart = tile.gridX + 1;
+        var rowEnd = rowStart + tile.gridHeight;
+        var colEnd = colStart + tile.gridWidth;
+
+        return "grid-area: " + rowStart + " / " + colStart + " / " +
+            rowEnd + " / " + colEnd + ";";
+    };
+
+    function addStyleString(str) {
+        var node = document.createElement('style');
```

Comment:
in general this is not great cause it'll leave a style block around forever and it's hard to reason about hte css on the page. this might be better with jquery's `.css` command. let's offline tomorrow